index.html

```diff
@@ -6,11 +6,20 @@
         <style type="text/css">
             body { margin:0; }
         </style>
-        <script src="assets/phaser.min.js"></script>
+        <script>
+        var supportsWebGL = ( function () { try { return !! window.WebGLRenderingContext && !! document.createElement( 'canvas' ).getContext( 'experimental-webgl' ); } catch( e ) { return false; } } )();
+        if(supportsWebGL){
+            document.write("<script src=\"assets/phaser.min.js\"><\/script>");
```

Comment:
What I'm not understanding is why we would need to have a case where we don't import the phaser library, it just seems messy to me.